### PR TITLE
Add default values to cast

### PR DESCRIPTION
**Pull request by Jellyfrog**
Makes the SQL query correctly select id columns as int instead of string for most ajax table requests

Not a performance issue I think, but if it was the other way (string to int) we could missed indexed searches.

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
@Jellyfrog what is the benefit of this?

**Comment by Jellyfrog** [comment]
Because the sql query ends up treating the integers as Strings, which in mysql seems to not have any effect tho. But might in orders.   
And it it was a indexed column that was a string "1" for example, and you searched for `column=1` (integer) it would have missed the indexed search.  
Might be a problem in other databases, not sure. 
Tldr; to treat integers as integers

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-67-release-changelog-august-2020/13285/1
